apps/backend/test/mocks/classes/pos_deposit_mock.ts

```diff
@@ -1,34 +1,27 @@
 import { faker } from '@faker-js/faker';
-import { Payment } from './payment_mock';
-import { Transaction } from './transaction_mock';
-import { BaseData } from '../types/interface';
+import { PaymentMock } from './payment_mock';
+import { FileMetadata } from './../../../src/common/columns/metadata';
 import { MatchStatus, MatchStatusAll } from '../../../src/common/const';
-import { Ministries } from '../../../src/constants';
 import { POSDepositEntity } from '../../../src/deposits/entities/pos-deposit.entity';
-import { LocationEntity } from '../../../src/location/entities/master-location-data.entity';
+import { LocationEntity } from '../../../src/location/entities';
 import { PaymentMethodEntity } from '../../../src/transaction/entities';
 
 /*eslint-disable */
-export class POSDeposit extends POSDepositEntity {
-  program: Ministries;
-  location: LocationEntity;
-  transaction_date: string;
-  transaction_amt: number;
-  paymentMethod: PaymentMethodEntity;
-  status: MatchStatus;
+export class POSDepositMock extends POSDepositEntity {
   constructor(
-    baseData: BaseData,
-    payment: Payment,
-    transaction: Transaction,
+    location: LocationEntity,
```

Comment:
I'd love a quick primer on why this changed so much

Reply:
For sure! Let's chat tomorrow. This was sort of experimental. I just wanted to be able to create some dummy-test-data quickly/easily instead of writing it out per unit test. But, it might too confusing. I'm open to changing/removing :)